subsys/bluetooth/host/Kconfig

```diff
@@ -48,16 +48,16 @@ config BT_RX_BUF_LEN
 	  (4 bytes) and the ACL header (also 4 bytes) which yields 73 bytes.
 
 config BT_HCI_TX_STACK_SIZE
-	int
+	int "HCI TX stack size"
 	default 512 if BT_H4
 	default 512 if BT_H5
 	default 416 if BT_SPI
-	default 940 if BT_CTLR && NO_OPTIMIZATIONS
-	default 640 if BT_CTLR
+	default 940 if BT_CTLR && BT_LL_SW && NO_OPTIMIZATIONS
+	default 640 if BT_CTLR && BT_LL_SW
 	default 512 if BT_USERCHAN
 	# Even if no driver is selected the following default is still
 	# needed e.g. for unit tests.
-	default 512
+	default 1024
```

Comment:
Did you check if this is sufficient when running a central example, i.e. central_uart?

Reply:
Running `zephyr/samples/bluetooth/central` with stack size 1024 seems to work fine. Though I discovered that when choosing BT_LL_NRFXLIB, the stack size isn't returned to 1024 after all :crying_cat_face: It seems that once a config is set it will not return to the default... I'll have to do some more testing/investigation on this then

Reply:
Apparently, by not having a prompt Kconfig will restore the final default value when no other matches their conditions. I've force pushed a clean commit with that now.